undertone-htb.js

```diff
@@ -306,84 +310,116 @@ function UndertoneHtb(configs) {
          *
          */
 
-        /* ---------- Process adResponse and extract the bids into the bids array ------------*/
-
         var adResponseDic = {};
-        if (adResponse != null) {
-            for (var i=0; i<adResponse.length; i++) {
-                adResponseDic[adResponse[i].bidRequestId] = adResponse[i];
+        if (adResponse !== null) {
```

Comment:
The `null` here needs to be updated to `undefined` to ensure we are comparing the type and value appropriately.

Reply:
I can't do that because when there's your lint check does not allow to write undefined